docs.md

```diff
@@ -74,35 +110,69 @@ Status: 200 OK
 
 Name | Type | Description
 ---|---|---
-name | string | **Required**. The name of the event.
-category | array of strings | **Required**. The event category / categories.
-accessibilityOptions | array of strings | Event accessibility options.
+ownerId | mongoose ObjectId | id of event owner.
```

Comment:
Why are we calling this a mongoose ObjectId, could we call it an id instead.Too me this seems like an implementation detail.